zips/zip-0271.md

```diff
@@ -147,7 +190,91 @@ to
+> defines a one-time disbursal of funds from this lockbox in order to support the
+> Community And Coinholder Funding Model [[ZIP-1016]](zip-1016).
+
+Change:
+
+> Consensus rule: [Canopy onward] In each block with coinbase transaction $\mathsf{cb}$
+> at block height $\mathsf{height}$, for each funding stream $\mathsf{fs}$ active at
+> that block height with a recipient identifier other than $\mathsf{DeferredPool}$ given
+> by $\mathsf{fs.Recipient}(\mathsf{height})$, $\mathsf{cb}$ MUST contain at least one
+> output that pays $\mathsf{fs.Value}(\mathsf{height})$ zatoshi in the prescribed way
+> to the address represented by that recipient identifier.
+
+to
+
+> Consensus rule: [Canopy onward] In each block with coinbase transaction $\mathsf{cb}$
+> at \block height $\mathsf{height}$, $\mathsf{cb}$ MUST contain at least one distinct
+> output for each of the following:
+>
+> * for each funding stream $\mathsf{fs}$ active at that block height with a recipient
+>   identifier other than $\mathsf{DeferredPool}$ given by $\mathsf{fs.Recipient}(\mathsf{height})$,
+>   an output that pays $\mathsf{fs.Value}(\mathsf{height})$ zatoshi in the prescribed way
+>   to the address represented by that recipient identifier.
+> * [NU6.1 onward] if the block height is $\mathsf{ZIP271ActivationHeight}, an output
+>   that pays $\mathsf{LockboxDisbursementAmount}$ zatoshi in the prescribed way to the
+>   Key-Holder Organization P2SH multisig address specified in [[ZIP-271]](zip-271).
+\end{itemize}
+
+Change the second bullet point defining the prescribed way to pay a Sapling payment
+address so that it also applies to Orchard, i.e.:
+
+> * The prescribed way to pay a Sapling or Orchard payment address is defined in
+>   [[ZIP-213]](zip-213), using the post-Heartwood consensus rules specified for
+>   Sapling and Orchard outputs of coinbase transactions in
+>   § 7.1.2 ‘Transaction Consensus Rules’.
+
+Clarify the notes as follows:
+
+> * The funding stream addresses are not treated specially in any other way, and
+>   there can be other outputs to them, in coinbase transactions or otherwise.
+>   In particular, it is valid for a coinbase transaction to have other outputs,
+>   possibly to the same address, that do not meet the criterion in the above
+>   consensus rule, as long as there is at least one distinct output that meets
+>   it for each funding item.
+> * For clarification, if there are multiple active funding streams or lockbox
+>   disbursements with the same recipient identifier and/or value, there MUST be
+>   at least one distinct output for each of them.
+> * Up to and including NU6.1 there have been no funding streams or lockbox
+>   disbursements defined with a shielded payment address as a recipient.
+>   That might change in future, so implementations are encouraged to support
+>   Sapling and Orchard outputs as recipients, as permitted by [[ZIP-213]](zip-213)
+>   and [[ZIP-207]](zip-207).
```

Comment:
ZIP 207 does not currently support this. If we want to support it, this should be made as a change in a separate PR. Remove these changes from this PR.